src/activeobject.h

```diff
@@ -66,8 +59,6 @@ enum ActiveObjectCommand {
 	AO_CMD_SET_BONE_POSITION,
 	AO_CMD_ATTACH_TO,
 	AO_CMD_SET_PHYSICS_OVERRIDE,
-	AO_CMD_OBSOLETE1,
```

Comment:
You've broken compat here by changing the value of future enum cases, you need to leave this in